pipes/Broad_UGER/jobscript.sh

```diff
@@ -2,11 +2,14 @@
 # properties = {properties}
 # this is identical to the default jobscript with the exception of the exit code
 
+source /broad/software/scripts/useuse
+use Python-3.4
```

Comment:
except, how would we know where to find the dotkit setup script, since we need $BINDIR first?